### PR TITLE
Add bors tool

### PR DESCRIPTION
**Pull request by alallema**

### CODE REVIEW

**Comment by curquiza** [comment]
bors try

**Comment by curquiza** [comment]
Thaaaaanks!!

Please add this badge: <a href="https://app.bors.tech/repositories/34949">Bors enabled</a>

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
 <p align="center">
   <a href="https://github.com/meilisearch/meilisearch-aws/blob/main/LICENSE"><img src="https://img.shields.io/badge/license-MIT-informational" alt="License"></a>
+  <a href="https://app.bors.tech/repositories/34948"><img src="https://bors.tech/images/badge_small.svg" alt="Bors enabled"></a>
```
Comment (curquiza) [inline README.md:18]:
```suggestion
  <a href="https://app.bors.tech/repositories/34949"><img src="https://bors.tech/images/badge_small.svg" alt="Bors enabled"></a>
```

**Review by curquiza — APPROVED** [review]
🎉

**Comment by alallema** [comment]
bors merge

**Comment by curquiza** [comment]
Looks like the `build-image-test` fails sometine

**Comment by alallema** [comment]
> Looks like the `build-image-test` fails sometine

Yes in digital-ocean too. I will open issue about it

**Pull request merged**